R/pkg/R/functions.R

```diff
@@ -207,7 +208,7 @@ NULL
 #' tmp <- mutate(df, v1 = create_array(df$mpg, df$cyl, df$hp))
 #' head(select(tmp, array_contains(tmp$v1, 21), size(tmp$v1)))
 #' head(select(tmp, array_max(tmp$v1), array_min(tmp$v1)))
-#' head(select(tmp, array_position(tmp$v1, 21), array_sort(tmp$v1)))
+#' head(select(tmp, array_position(tmp$v1, 21), array_repeat(21, 5L), array_sort(tmp$v1)))
```

Comment:
this example is a bit unusual? do you intend for the first param to be `21` the constant? (also, does that work?)

Reply:
also for `5L`, `5` should be ok and more clear as well